file_utils.sh

```
@@ -1,12 +1,11 @@
-
-
+source "$(dirname "$0")/config.sh"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)